cql3/statements/select_statement.cc

```diff
@@ -1626,11 +1626,15 @@ std::unique_ptr<prepared_statement> select_statement::prepare(data_dictionary::d
     // using `forward_service`.
     auto can_be_forwarded = [&] {
         return selection->is_aggregate()        // Aggregation only
-            && selection->is_count()            // Only count(*) selection is supported.
+            && ( // SUPPORTED PARALLELIZATION
+                selection->is_count()
+                || selection->is_reducible()
+            )
             && !restrictions->need_filtering()  // No filtering
             && group_by_cell_indices->empty()   // No GROUP BY
             // All potential intermediate coordinators must support forwarding
             && db.features().parallelized_aggregation
+            && db.features().full_parallelized_aggregation
```

Comment:
Ah, it's here, but it's wrong, since now a partially upgraded cluster will lose the previous support for count parallelization. It should apply only to the new feature.